src/program/snabbvmx/lwaftr/setup.lua

```diff
@@ -17,6 +17,7 @@ local nh_fwd = require("apps.lwaftr.nh_fwd")
 local pci = require("lib.hardware.pci")
```

Comment:
Is there a reason this doesn't just slightly extend the lwaftr's setup.lua? It's 440 lines (to the main lwaftr's setup.lua's 391), which seems huge.